### ISSUE TITLE
warning in base_riskscore

### ISSUE THREAD

**Issue by benkeser** [CoVPN/correlates_reporting#207]
In [a recent build](https://travis-ci.com/github/CoVPN/correlates_reporting/builds/221617265), I am seeing the following warning (line 1435 of build log) when `code/tables_figures.R` is executed:

````
Warning message:
Problem with `mutate()` input `Learner`.
ℹ Unknown levels in `f`: SL.ranger
ℹ Input `Learner` is `fct_relevel(...)`. 
````

@brborate can you confirm whether this warning is OK to ignore? The report still builds, but not clear if it's as expected or not. For verification purposes, I can't look into the issue further myself.

**Comment by brborate** [comment]
updated this, will add it in the current PR.

**Comment by nhejazi** [comment]
Which was the PR in which this was addressed? Was it #215?

**Comment by brborate** [comment]
#215 only caters to xgboost version.  Rest of the changes requested by David will be addressed in upcoming update, mostly by tomorrow.
